example/controller-registration.yaml

```diff
@@ -1,18 +1,16 @@
3g2x0kQCvJBC6M+N2/X8DwrBzv13JjScsoALSY+ksWv/cdtX979Zb9Vx/+tfdKUPwJ98/7+FAdGaSq5AC0i3GeYTymEUs9BnPICIeHckoMqxvoW3E6ZAxVEkpMYGqkYIQShGMCXam+Do1yBpSDSbUZynJ6V+wn1EwGmATwWHl5GkY/aB+jBnOO6bVw5c83ABgiczDUsQUQkh49SxnMvh+6FG3hDFhZhOEcHtxRB8JpXlBEy7yTVl33JGH6WbXPOOSeCaS36rZtxdIhrh+uIIxiykyvreUfMIryNyh1c9xfZ/cegtkUzECnqXXSQYSfEb9bTlMJ8SNx2HXZYzU57wqWs9967uD7vt/2JCpHYWZBoeS2OX/TdPNuy/ed6q7P8pgETslkqz722YNSwSRaVbp+7UbZ/OLJ8qT7JIJ/0deIMRATyjFzAWEvSEwk+ZHkFwa7QGCnWyOJnSNuzUM2u2he5zS+frh9327wvPCcRjaOywf4z2jbX87/z0tFHZ/1OA62IYjBYYKScaXnqvoFlv/BWGnQEMu4B2THhyQ8YYHhnRFDwxjQhfONDB0J9MUxjyFZUz6jtpfmAiKeBvyDxUKYzwMfdp6iY6mEzgz1CM9ZxgpnGVDnkNMwea6DQ8GmkgCrjQOE/gFDlnCrHxZPpV76LbR8YMBct18S/HsIVIgTvzaNB06vDSDKhlj2qv/mZQLESMecrCEIUYieliERlDSN0sGwXAPZrmK3pJwDE43mU4xEgTHE5wQoR34/JAIDpjOoGJ1lHbdefzuUMSjh0hAzcTmnKztdrIdTbrF44ZipH2f2ImccWjBaC/xglkhLyGZJ5sWCApPjPJHIe5xKTIJF8qE7hB4zOlJRvFekVoOY+49PIAFBuqQK0zhN6wBj90hr3ha4Pk197bN9e/vIVfOzc3nf7bXncI1zdwcd2/7L3tXffx7kfo9N/BP3v9y9dAmdlJFCcmfbgCZJMZcaLGGFxDSldYyGOKiqjHxszDpfEgxhQUAoFRgidJKZVTpsy2qiSzRDQhmzKdJJdqc12OhUMC0Q6MszN67Dgu/s0o99HHYe44iUcObrKbO0R30zMqd4KpopujsD3BtRRhiGMkDYzQEuqOmsCqDwUnI3fbvRmidLI7+oGgFKj7EB6Tc6F1cqNRVN6kKNtZuDQrHqSZeRaKKTd6oKC8yixVT0SadRrpGcF4QkpMYmFJHVaoW1EZexWKv07YHf81RSVFXVJHnwQdfP7TrJ+cNqvzn6eAQ/b/PZb76HeVo6ODasFd+3/aWq//mmfn1fnPk8D9vQ3g0zFGGKixKfr7GtifPlkA5gkbw4SoQXJSAzU1Ic3WWbsGzi0JY6qcZLyjSQDFjEgyrsdQ+0794zu1PlLSSCimhVx8DgUNMQfYgrB9NELum5tSM2nnq/ZpFIrFlHKdlaCpBDC1Ui6WxPk00/fcu/Xl4RD7XwrqsOOgXfVf62yt/mviTXX+8yRQPv/JNf6Ocb8Nl8VuW1OqiU80aaNB7Xuck45UmETi8Pt7cG5oSAkm4v28OzXQkIzQ4A1mMAw4d/EIU3yqjSkLd39qWE/RcIq5t5vkq/tO2iTJOGoE38a1YdjUJIZZSREH4nyDCTO6nytTe7ShnjxJSjKVzs+8UtZ5IZCDdN0KEXs4NV15ckx+VRLFY4Vx+MoAckvPWCptuoFwhbvH83cMhwC5/JM2lTOs7jqeZ8TaP4i6l5dVxYrsA1Q7hSTUJKwy7oWxv4ygTs5uMWwQh+FAoA4sVvQijVZR8bA8D6vRKZa2S4nbsKUofYg5G2x7Sj6YItWLsdLjGotLc2Nec/y9xMKy/kva2eDhgnuqzI3Bl74/sUVE0xrRJpyLtOR+CGM65Tqf0SkmlHFTPiuvMt2Hq27nsnvzvnvVvTAnCu/7nZ+7w0HnoluMBJgZgj9KMW2XOgHGjIb+DR2v9mb9A6In7UK7ncJLWVnCszRZJWLp0RUpFJ1tHA5avDMl+uaM34GjD0UxQqO+lno8t8Nfg0PivxwR75gXQTvi/2mjfr6e/zdM/VfF//8/2LZtlXOAZI9JrCdCso/pQdDdXxLfXCQGFyHKjMobEdIjM4M/WMyXcWgM3saJ7Ccp4ihj3V6+5VJOTtrxQhH71oqrSAd7qdwU3mKhMSr6A6qzVojJRNacm4RgneQBBO1lhMskoLb3urhyHa8wlbNUMJSzY1pR0Yoj3Hm6yWOttsmMop6kWpXbLlZfJGQf1yTyWeI40ZAsUV+ytBcfuPwxC6YkSlmhM/TRa+QzGoejy58lCW76fFX5bFQr3yinyfzMq9Gty56vC/jBJSYUcob2UEDMJjDb8Iv+vRSxELm58ZH14iZntKwVD+tFIULTMJVG1hyVZj7ofPJpfnbmv3JQvTokYiUbKR5tyKII+zkfhSqsquvGLidjuR8Jthy+LIyXCJLUdO2WpJlqgTd1CJgq0S1dI3S2jAf5k/KoLY9/E6O8GQm/3HRDEeQKEOvk/H9ORxMh7tJ1xakYCw6QATHN5ZWckbD8+Yq21L6vWY+LHj+kS/iTBBFccJaS5nbxGXnhqM1gu790VDwyHyglQSvFM1yplb54Hf/caUwFR8Ih+f+qF9u/Eth9/r+e/582W9X5/5PA2vdfW51FdfxXePHn3q4vDofY/ywiR30HutP+z5vr9T/+Vfb/FLCewdUwGxHKw8KMB8vcbYQeoFnLvAOO1QxHDITfyQZTeYyTsFGh9nQU+bEv7hVWKcXR3ra3F6Z/4w2G6dzfmaQVTHZcm8xNe34WPmKomVXXvhJXsNv+Z+nh5iM+AN9l/yfr3382662T6vvPJ4H0HUZSIeev0ttAYyfwpDH9Qj2yf3lYfh5X+ipuw340CdqQxAxjeVHp1Udv3Bd6YL4XRaO0llU53H+yrNJ7A8PR2suINrSw88H3CW0Yk1BRy/KZMt/BXZSQwb/+/ZWYawUVVFBBBRVUUEEFFVRQQQUVVFBBBRVUUEEFFVRQwU74H/64Q84AUAAA
```

Comment:
ImagePull error when I am trying to use this ControllerRegistration. 
eu.gcr.io/gardener-project/gardener/extensions/runtime-gvisor:v1.0.0-dev. does not exist.
I guess you used the dockerhub image for testing?